com.unity.formats.alembic/Runtime/Unity.Formats.Alembic.Runtime.api

```diff
@@ -56,7 +56,7 @@ namespace UnityEngine.Formats.Alembic.Importer
         public AlembicPointsRenderer() {}
     }
 
-    [UnityEngine.ExecuteInEditMode] public class AlembicStreamPlayer : UnityEngine.MonoBehaviour
+    [UnityEngine.DisallowMultipleComponent] [UnityEngine.ExecuteInEditMode] public class AlembicStreamPlayer : UnityEngine.MonoBehaviour
```

Comment:
shouldn't attributes be on separate lines?

Reply:
This file is autogenerated by the coding style package (also the whole code is auto- indented by the same package )

Reply:
ah ok, nvm then